Makefile.am

```diff
@@ -14,6 +13,7 @@ noinst_HEADERS = include/common.h \
 
 lib_LTLIBRARIES = libbzip3.la
 libbzip3_la_SOURCES = src/libbz3.c
+libbzip3_la_LDFLAGS = -version-info 0:0:0
```

Comment:
I'm a little out of my depth there so this is a question not a challenge ... is there a use case for `0:0:0` here or would a single versioning field cover what this project is likely to need in the form or `0`?